src/etc/inc/system.inc

```diff
@@ -1504,6 +1504,12 @@ EOD;
 		$nginx_config .= "\t\tadd_header X-Content-Type-Options nosniff;\n";
 		$nginx_config .= "\t\tssl_session_tickets off;\n";
 		$nginx_config .= "\t\tssl_dhparam /etc/dh-parameters.4096;\n";
+		if ($config['system']['webgui']['webgui-ocsp-staple'] !== false) {
+			$nginx_config .= "\t\tssl_stapling on;\n";
+			$nginx_config .= "\t\tssl_stapling_verify on;\n";
+			$nginx_config .= "\t\tresolver 8.8.8.8 1.1.1.1 valid=300s;\n";
```

Comment:
The resolver addresses should not be hardcoded, and definitely shouldn't default to 8.8.8.8 and 1.1.1.1. There should be configured DNS servers to use in the system general settings. Or failing that, 127.0.0.1 so it will use the DNS Resolver or Forwarder. Basically, use the same logic that is used to create resolv.conf.

Reply:
Sounds smart. I'll hunt around for that logic and change this.

Reply:
Added.